### PR TITLE
ci: update lint and test failure cases

### PR DESCRIPTION
**Pull request by ccalbaugh**
This makes sure jobs don't fail silently

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Branch being merged does not follow [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>You should add at least 1 more reviewer to the PR</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b73e0c98dc31908e5401fe7b1d33254382fc2ea0
</p>

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -6,29 +6,31 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
```
Comment (rudolph9) [inline .github/workflows/pull-request.yaml]: Why did you change the indentation?

Comment (ccalbaugh) [reply .github/workflows/pull-request.yaml]: That was done automatically by the code formatter.  I'll un-indent it

**Review by rudolph9 — APPROVED** [review]
